app/src/main/res/menu/menu_bookmarks.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <menu xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
```

Comment:
Reformat using Square's code style `tools` will be below `app`.

Reply:
@abdulwd Changes done. Please review.